ocaml/xapi/xapi_config.ml

```diff
@@ -37,7 +37,13 @@ let read_config filename =
 		"igd-passthru-vendor-whitelist", Config.Set_string vendor_whitelist;
 		"relax-xsm-sr-check", Config.Set_bool Xapi_globs.relax_xsm_sr_check;
 		"hotfix-fingerprint", Config.Set_string Xapi_globs.trusted_patch_key;
-		"gpumon_stop_timeout", Config.Set_float Xapi_globs.gpumon_stop_timeout;
+		(let cgo = "ciphersuites-good-outbound" in
```

Comment:
No, we definitely need to keep the `gpu_stop_timeout` line.
My original commit only added the ciphersuites-related lines here, but when you applied my change to a different base you accidentally removed the `gpu_stop_timeout` line.

The commit 699ae79d6a1169754abaf4eee1090143dc491f24 that you quote above has me as "author" but was committed by you, as a change to code that included the `gpumon_stop_timeout` line.

My original commit added some lines in that chunk but did not remove `gpumon_stop_timeout` because it was not there to start with. This was commit 25a99e89f047abbd6d4a4a576c4f2d0ce4302628 on my own branch: see https://github.com/xenserver/xen-api/pull/68/commits/25a99e89f047abbd6d4a4a576c4f2d0ce4302628